### ISSUE TITLE
NTR: Neuraminidase 1 low in fibroblasts or leukocytes (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8133]
New term request Neuraminidase 1 low in fibroblasts or leukocytes (f387): alpha neuraminidase

**Comment by pnrobinson** [comment]
@MickeySegal
I think the enzyme activity that is measured is just Neuraminidase  (not "1") 
https://enzyme.expasy.org/EC/3.2.1.18
What is the context?

**Comment by MickeySegal** [comment]
OMIM indicates that these are the same thing, but currently uses the form without the number:  https://omim.org/entry/256550.  but the gene still has the number: https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:7758

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000911
- label: Reduced tissue neuraminidase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:610423
- def: Activity of neuraminidase (EC 3.2.1.18)in tissues below the lower limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts. Neuraminidase is also known as lysosomal sialidase.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8133
- synonyms:
